pkg/blockstorage/helpers.go

```diff
@@ -33,8 +33,11 @@ const (
 	AzureCientID               = "AZURE_CLIENT_ID"
 	AzureClentSecret           = "AZURE_CLIENT_SECRET"
 	AzureResurceGroup          = "AZURE_RESOURCE_GROUP"
+	AzureResurceMgrEndpoint    = "AZURE_RESOURCE_MANAGER_ENDPOINT"
```

Comment:
Should we add the new configs in this PR to https://github.com/kanisterio/kanister/blob/ab6010860929c1702fb94c238607a6115bcc621a/pkg/blockstorage/azure/client_test.go#L39 ? The test is currently being skipped . File a ticket to track  it ?

Reply:
Yes, all azure tests are skipped as of now. Will search if a ticket exists or will file one

Reply:
👍🏼